doc/website/concepts/architecture.md

```diff
@@ -1 +1,64 @@
 # Architecture
+
+This article provides an overview of the Eclipse iceoryx architecture and explains the rationale for it.
+
+## Software layers
+
+The main packages which Eclipse iceoryx is composed of are depicted below.
+
+![Software layers](iceoryx_software_layers_v2_0_0.svg)
```

Comment:
The links are broken, please fix them and use `../website/images/iceoryx_software_layers_v2_0_0.svg`

Please update this for all links.

Reply:
@mossmaurice would `../images/iceoryx_software_layers_v2_0_0.svg` work? This is different than the hyperlinks since the rendered markdown file looks broken

Reply:
If we don't ever change the position of the images and they'll always be in this folder, I'm ok with relative links for images. Adapted.